lib/generators/templates/ci/ci.yml.tt

```diff
@@ -0,0 +1,146 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+<%- if Bundler.rubygems.find_name("bundler-audit").any? -%>
+  scan_ruby:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+          bundler-cache: true
+
+      - name: Scan for security vulnerabilities in Ruby dependencies
+        run: |
+          bin/rails bundle:audit:update
+          bin/rails bundle:audit
+<% end -%>
+
+<%- if File.exist?("bin/importmap") && File.exist?("package.json") -%>
+  scan_js:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+          bundler-cache: true
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .node-version
+
+      - name: Install modules
+        run: yarn install
+
+      - name: Scan for security vulnerabilities in JavaScript dependencies
+        run: |
+          bin/importmap audit
+          yarn audit
+<% end -%>
+
+<%- if Bundler.rubygems.find_name("standard").any? && File.exist?(".prettierrc") -%>
```

Comment:
Checking for `.prettierrc` doesn't guarantee that  `yarn lint` exists. I wonder if we want to parse `package.json`?